src/app/lists/ListDataview.tsx

```diff
@@ -19,49 +14,81 @@ import TooltipCell from "@/app/lists/TooltipCell";
 import TableSurface from "@/components/Tables/TableSurface";
 import CommentBox from "@/app/lists/CommentBox";
 
-export type ListRow = Schema["lists"] & Datum;
+interface ListRow {
+    itemName: string;
+    "1": QuantityAndNotes;
+    "2": QuantityAndNotes;
+    "3": QuantityAndNotes;
+    "4": QuantityAndNotes;
+    "5": QuantityAndNotes;
+    "6": QuantityAndNotes;
+    "7": QuantityAndNotes;
+    "8": QuantityAndNotes;
+    "9": QuantityAndNotes;
+    "10": QuantityAndNotes;
```

Comment:
Is it "10+" or "10" because it is 10 or more right

Reply:
The database naming scheme calls it `10_(quantity|notes)` and these keys aren't used for rendering so I think it's better to stick with this.

Reply:
yeah I agree (and I think similarly for the ones below as well)